src/noaa_apt.rs

```diff
@@ -150,28 +151,41 @@ pub fn process(
         }
     };
 
+    // for colorization & histogram equalization,
+    // always do 98% contrast adjust first, then colorize,
+    // then equalize histogram of color image if needed
+    if color.is_some() {
+        if let Contrast::Histogram = contrast_adjustment {
+            let (l, h) = misc::percent(&signal, 0.98)?;
+            low = l;
+            high = h;
```

Comment:
```suggestion
            (low, high) = misc::percent(&signal, 0.98)?;
```

Reply:
Unfortunately destructuring assignments are not yet supported: https://github.com/rust-lang/rfcs/issues/372 so this is a workaround.